test/acceptance/behave/components/common_steps/requests.py

```diff
@@ -280,6 +308,7 @@ def verify_that_receive_several_http_codes(context, http_code):
     __logger__.info("...Verified that http code returned in all entities are %s" % http_code)
 
 
+@step(u'verify that entities are stored in default mongo')
```

Comment:
default mongo -> default tenant

or

default mongo -> default tenant at mongo

as "default mongo" could be missleading (it could be interpreted as the "default mongo *database server*").

This change should be done in the overall .feature, not only in this specific point.